staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -623,6 +629,15 @@ func (e *Store) Get(ctx genericapirequest.Context, name string, options *metav1.
 	return obj, nil
 }
 
+// qualifiedResourceFromContext retrieves GroupResource from Context per request, if not found, return DefaultQualifiedResource
+func (e *Store) qualifiedResourceFromContext(ctx genericapirequest.Context) schema.GroupResource {
+	if info, ok := request.RequestInfoFrom(ctx); ok {
+		return schema.GroupResource{Group: info.APIGroup, Resource: info.Resource}
+	}
+	// some lister implemented accesses the storage directly and it doesn't have a RequestInfo
```

Comment:
some implementations access storage directly and thus the context has no RequestInfo